editor/editor_interface.h

```diff
@@ -35,6 +35,7 @@
 #include "core/object/class_db.h"
 #include "core/object/object.h"
 #include "core/object/script_language.h"
+#include "editor/editor_feature_profile.h"
```

Comment:
I don't see a reason to have this in the header. Should be moved to the cpp file.

Reply:
Oops, change left over from testing. Nice catch 😉